testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -783,4 +785,34 @@ public void verifyFT77()
                 expectedJsonPattern, ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Ignore
```

Comment:
Couple of questions - why is this marked ignore? Do we already have a jira ticket for the failures? Can you add it here, if that's true.

Also, it is usually good practice to have a test case per scenario to make that they are aren't cascading failures and it is easy from a failure to pinpoint a particular scenario than mixing them up